functions/license_uuid.pp

```diff
@@ -0,0 +1,12 @@
+# return the uuid from a Puppet license file supplied in $content
+# If no $content parameter specified, tries to read the license file
+# from /etc/puppetlabs/license.key
+function rsan::license_uuid(Optional[String] $content) >> String {
```

Comment:
Looks Great,

Am I right in saying this would only work locally on a server that has the licence, if i wanted to consume the UUID on a different server, is there a way to do that?

Reply:
@MartyEwings this function will work if the catalog is compiled on the PE primary. It can also be made part of the export logic for the primary so it writes a file shared over NFS

Reply:
Of course as long as the catalog is compiled on the primary, the uuid value in the catalog can be used on any other machine